pkg/pb/gracefulexit.proto

```diff
@@ -77,22 +78,22 @@ message StorageNodeMessage {
 message NotReady {}
 
 message TransferPiece {
-  bytes piece_id    = 1 [(gogoproto.customtype) = "PieceID", (gogoproto.nullable) = false]; // the current piece-id
+  bytes piece_id    = 1 [(gogoproto.customtype) = "PieceID", (gogoproto.nullable) = false]; // the original piece ID
```

Comment:
Rename the field to `original_piece_id` rather than using comments for it.

Reply:
Updated